src/lib/components/common/RichTextInput.svelte

```diff
@@ -983,7 +993,7 @@
 				StarterKit.configure({
 					link: link
 				}),
-				Placeholder.configure({ placeholder: () => _placeholder }),
+				Placeholder.configure({ placeholder }),
```

Comment:
This commit appears to contain other commits from the dev branch and may not have been rebased correctly.

We should not use `merge` to incorporate changes from the latest dev branch. Instead, use `git rebase dev` or `git rebase upstream/dev` (if your upstream source is the original repository).